scripts/build-desktop.sh

```diff
@@ -24,4 +24,6 @@ info ""
 title "Building Desktop App Archive Using Electron"
 info ""
 shift 1
-"$LOCAL_PACKAGES/electron-builder" --config config/electronBuilder.config.js "$@"
+"$LOCAL_PACKAGES/electron-builder" --config config/electronBuilder.config.js --x64 "$@"
```

Comment:
Can we refactor this to run both builds in parallel in the background? I did something similar with the `.github/scripts/buildActions.sh` that you could use as a reference if needed.

Reply:
Okay, thanks for looking into it.